03-vectors/vectors-practice-2.Rmd

```diff
@@ -68,6 +79,18 @@ Let's take a look at the second example:
 
 ```{r}
 draw(p2x, p2y)
+
+
+new.p2x <- p2x
+new.p2y <- p2y
+
+new.p2x[new.p2x > 50 & new.p2y > 50 ]<- new.p2x[new.p2x > 50 & new.p2y < 50  ]
```

Comment:
It looks like you're trying to "copy" the points of one of the Hobbes into the empty quadrant, which is a cool strategy...